x-pack/plugins/security_solution/cypress/e2e/exceptions/alerts_table_flow/endpoint_exceptions.cy.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  goToClosedAlertsOnRuleDetailsPage,
+  openAddEndpointExceptionFromFirstAlert,
+} from '../../../tasks/alerts';
+import { deleteAlertsAndRules } from '../../../tasks/common';
+import { login, visitWithoutDateRange } from '../../../tasks/login';
+import { getEndpointRule } from '../../../objects/rule';
+import { goToRuleDetails } from '../../../tasks/alerts_detection_rules';
+import { createRule } from '../../../tasks/api_calls/rules';
+import {
+  waitForAlertsToPopulate,
+  waitForTheRuleToBeExecuted,
+} from '../../../tasks/create_new_rule';
+import {
+  esArchiverLoad,
+  esArchiverResetKibana,
+  esArchiverUnload,
+} from '../../../tasks/es_archiver';
+import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../../urls/navigation';
+import {
+  addExceptionFlyoutItemName,
+  selectCloseSingleAlerts,
+  submitNewExceptionItem,
+} from '../../../tasks/exceptions';
+import { ALERTS_COUNT, EMPTY_ALERT_TABLE } from '../../../screens/alerts';
+import {
+  EXCEPTION_ITEM_CONTAINER,
+  FIELD_INPUT_PARENT,
+  NO_EXCEPTIONS_EXIST_PROMPT,
+} from '../../../screens/exceptions';
+import {
+  removeException,
+  goToAlertsTab,
+  goToEndpointExceptionsTab,
+} from '../../../tasks/rule_details';
+
+describe('Endpoint Exceptions workflows from Alert', () => {
+  const expectedNumberOfAlerts = 1;
+  before(() => {
+    esArchiverResetKibana();
+    esArchiverLoad('endpoint');
+    login();
+    deleteAlertsAndRules();
```

Comment:
If we use `cleanKibana()` we don't need the `deleteAlertsAndRules()` method because the cleanKibana() includes it.

Reply:
I believe if we are using the `esArchiverResetKibana` we don't need the `deleteAlertsAndRules` as well, that's why I removed them from most of the tests as well